magefile.go

```diff
@@ -334,3 +337,13 @@ func PythonEnv() error {
 	_, err := mage.PythonVirtualenv(true)
 	return err
 }
+
+func BuildOpaBundle() error {
```

Comment:
I'd rather not have two different ways to build a bundle. (magefile and justfile).
This can cause problems if in the future someone only change one of them and they diverge.
What about consolidating the bundle building logic to a single place and then using it from wherever needed?

Reply:
Good catch, Our build scripts are a bit messy, working on consolidation. As a temporary step invoking mage command from justfile is a good solution.